File: TODOS EXERCICIO DE PYTHON/ex35.py
''' André Cavalcante de Lima
Armazenar vinte valores em um vetor. Após a digitação, entrar com uma constante multiplicativa 
 que deverá multiplicar cada um dos valores do vetor e armazenar o resultado no próprio vetor, 
 na respectiva posição.
'''
numeros = []

for i in range(0, 20, 1):
    x = int(input('Digite um numero: '))
    numeros.append(x)

m = int(input("Digite a constante multiplicativa: "))

for i in range(0, 20, 1):
 ##   numeros[i] = numeros[i]*m
    print( numeros[i], " X ", m, " = ", numeros[i]*m)
   

File: October4thC 33-40/ex35.c
/* André Cavalcante de Lima
35. Armazenar vinte valores em um vetor. 
Após a digitação,
 entrar com uma constante multiplicativa que deverá multiplicar cada um dos valores do vetor 
 e armazenar o resultado no próprio vetor, na respectiva posição.

*/
int main(int argc, char const *argv[])
{
    // Declarando o vetor com 20 posições
  int n[20]; 
  int c;
    // Looping para popular o vetor
  printf("Digite os 20 numeros:\n");
  for(int i=0; i<20; i++){
      scanf("%i", &n[i]);
}
  // Solicita entrada da constante de multiplicação 
  printf("Digite a consante de multiplicacao: ");
  scanf("%i", &c); 

  // Looping para multiplicar os valores do vetor
  printf("Os numeros digitados foram: ");
  for(int i=0; i<20; i++){
      n[i]= n[i]*c; 
      printf("%i \n", n[i]);
     
  }




    return 0;
}